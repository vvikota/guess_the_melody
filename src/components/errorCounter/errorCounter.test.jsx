import React from 'react';
import renderer from 'react-test-renderer';
import ErrorCounter from './errorCounter.jsx';

it(`ErrorCounter is correctly render`, ()=> {
  const tree = renderer
  .create(<ErrorCounter
    mistakes={3}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
