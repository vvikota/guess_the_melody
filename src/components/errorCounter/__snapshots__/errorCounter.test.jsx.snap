// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorCounter is correctly render 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
